packages/@sanity/form-builder/src/sanity/legacyPartImplementations/__tests__/form-builder.test.tsx

```diff
@@ -0,0 +1,376 @@
+              "_type": "conditionalFieldsTest",
+              "_updatedAt": "2021-11-05T12:34:29Z",
+              "fieldWithObjectType": Object {
+                "field1": "dqwdw",
+                "field2": "vdsvdsvdsnkjnjkk",
+                "roleConditionField": "dvsvd",
+              },
+              "hiddenTestTitle": "hide me",
+              "isPublished": true,
+              "readOnlyIfTitleIsReadOnly": "dweefbfdssnflewnfklfwnlekfss",
+              "readOnlyTestTitle": "read only",
+            },
+            "parent": Object {
+              "_createdAt": "2021-11-04T15:41:48Z",
+              "_id": "drafts.10053a07-8647-4ebd-9d1d-33a512d30d3a",
+              "_rev": "5hb8s6-k75-ip4-4bq-5ztbf3fbx",
+              "_type": "conditionalFieldsTest",
+              "_updatedAt": "2021-11-05T12:34:29Z",
+              "fieldWithObjectType": Object {
+                "field1": "dqwdw",
+                "field2": "vdsvdsvdsnkjnjkk",
+                "roleConditionField": "dvsvd",
+              },
+              "hiddenTestTitle": "hide me",
+              "isPublished": true,
+              "readOnlyIfTitleIsReadOnly": "dweefbfdssnflewnfklfwnlekfss",
+              "readOnlyTestTitle": "read only",
+            },
+            "value": undefined,
+          },
+        ],
+        Array [
+          Object {
+            "currentUser": null,
+            "document": Object {
+              "_createdAt": "2021-11-04T15:41:48Z",
+              "_id": "drafts.10053a07-8647-4ebd-9d1d-33a512d30d3a",
+              "_rev": "5hb8s6-k75-ip4-4bq-5ztbf3fbx",
+              "_type": "conditionalFieldsTest",
+              "_updatedAt": "2021-11-05T12:34:29Z",
+              "fieldWithObjectType": Object {
+                "field1": "dqwdw",
+                "field2": "vdsvdsvdsnkjnjkk",
+                "roleConditionField": "dvsvd",
+              },
+              "hiddenTestTitle": "hide me",
+              "isPublished": true,
+              "readOnlyIfTitleIsReadOnly": "dweefbfdssnflewnfklfwnlekfss",
+              "readOnlyTestTitle": "read only",
+            },
+            "parent": Object {
+              "_createdAt": "2021-11-04T15:41:48Z",
+              "_id": "drafts.10053a07-8647-4ebd-9d1d-33a512d30d3a",
+              "_rev": "5hb8s6-k75-ip4-4bq-5ztbf3fbx",
+              "_type": "conditionalFieldsTest",
+              "_updatedAt": "2021-11-05T12:34:29Z",
+              "fieldWithObjectType": Object {
+                "field1": "dqwdw",
+                "field2": "vdsvdsvdsnkjnjkk",
+                "roleConditionField": "dvsvd",
+              },
+              "hiddenTestTitle": "hide me",
+              "isPublished": true,
+              "readOnlyIfTitleIsReadOnly": "dweefbfdssnflewnfklfwnlekfss",
+              "readOnlyTestTitle": "read only",
+            },
+            "value": undefined,
+          },
+        ],
+      ]
+    `)
+  })
+})
+// todo: move tests to the individual input components
```

Comment:
@vicbergquist @hermanwikner can we remove these commented tests? 👀

Reply:
they have been removed in the latest version of our branch